python/cuml/svm/svc.pyx

```diff
@@ -320,6 +338,57 @@ class SVC(SVMBase, ClassifierMixin):
 
         return sample_weight
 
+    def _get_num_classes(self, y):
+        """
+        Determine the number of unique classes in y.
+        """
+        y_m, _, _, _ = input_to_cuml_array(y, check_cols=1)
+        return len(cp.unique(cp.asarray(y_m)))
+
+    @cuml.internals.api_base_return_any_skipall
+    def _fit_multiclass(self, X, y, sample_weight) -> "SVC":
```

Comment:
Generally, you can omit decorators for private functions (leading `_`) if you are using the "skip_all" variant and decorate the return type. Has the same effect.

Reply:
FYI, the reason you can omit the decorator is based off the assumption that all private methods will be called by a public one that is decorated. So all of the decorator fun stuff will happen on the public call first so decorating the private method is redundant. 

Some private methods do have decorators to allow 2 public methods to share the same functionality. So if a private method is responsible for setting `self._input_type` and `self.output_dtype`, you would want to decorate that and have any public methods use the `_skip_all` variants.

Just some extra info for the future. I'm glad to see the decorators used correctly by someone other than myself. Will add these bits of information to the estimator guide for others.

Reply:
That is good to know, I was not aware of these details. It would be definitely a useful addition to the estimator guide.